MAUI/Shimmer/overview.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: About .NET MAUI Shimmer control | Syncfusion
+description: Learn here about the overview of Syncfusion .NET MAUI Shimmer (SfShimmer) control, its basic features, elements and more.
+platform: maui
+control: SfShimmer
+documentation: ug
+---
+
+# Overview of .NET MAUI Shimmer (SfShimmer)
+
+Syncfusion .NET MAUI Shimmer (SfShimmer) control can increase an application's responsiveness by displaying a shimmer effect while the content is loading or processing in the background.
+
+## Key features
+
+* **Multiple Shimmer View types**: There are seven different Shimmer view types available in the .NET MAUI `SfShimmer`. The `CirclePersona` view is initially rendered by default.
+
+* **Multiple Shimmer Wave Directions**: It defines the Animation wave direction for the Shimmer. There are five wave directions available in the `SfShimmer`. By default, the wave direction is rendered by the `Default` wave, allowing the animation from the left top to the right bottom.
+
+* **Shimmer Custom view**: `SfShimmerView` supports `ShimmerView`, which can be used in Custom view to change the Shimmer shape types.
```

Comment:
`SfShimmerView` supports `ShimmerView` -- Check the meaning

Reply:
Checked and shimmer view content modified